packages/client/src/pages/FindAMentor/index.js

```diff
@@ -17,21 +18,22 @@ const FindAMentorPage = () => {
 
     const { user } = useAuth();
     const { tags, setTags } = useState([]);
-    const { subjects, setSubjects} = useState([]);
+    const { subjects, setSubjects } = useState([]);
     const handleMentorshipRequest = (event) => {
         // TODO(johanc): Implement.
-        
+
     };
 
     const openModal = (mentor) => {
         // TODO(johanc): Implement.
     }
     const FilterColumn = () => {
         // TODO: Implement - should use setTags() to update the state.
-        <a onClick={() => setTags([])}>Placeholder filter component.</a>
+        // <a onClick={() => setTags([])}>Placeholder filter component.</a>
     };
 
-    const mentors = await get("/api/mentors", {tags: tags, subjects: subjects});
+    // const mentors = await get("/api/mentors", {tags: tags, subjects: subjects});
+    const mentors = [];
     const MentorColumn = () => {
         // A grid of mentor components.
         return (
```

Comment:
I think instead of making a `Modal` for every mentor, we should just use 1 modal for all the mentors. Would need to modify `Modal` so it can be controlled by a passed in prop (like `open` or something) as well as the current `trigger` prop. Then we can just have each Mentor component in the grid set the modal to open + populate it with its data.

we can do this in another PR or this one if you want to

Reply:
How would one modal work? I do not think I understand what you are saying. Isn't each mentor a modal that can be clicked to get more information and request?

Reply:
yeah I think to do this we might need to modify Modal a lot? right now the trigger is what needs to be clicked to render the modal, so I don't know if we can really set the Modal's data onclick if it doesnt know what it's listening to